moby/moby
# libnetwork/overlay: remove dead code

**- What I did**

All the code related to Classic Swarm is removed from the overlay driver, as well as all the code paths becoming dead because of that.

**- How to verify it**

CI should be green. It can also be tested by forming a swarm cluster with two dev containers:

```
dev1$ docker network create --driver overlay --scope global --attachable testnet
dev1$ docker run -d --rm --name whoami --network testnet traefik/whoami
dev2$ docker run --rm -t --name netshoot --network testnet nicolaka/netshoot curl http://whoami
```

**- A picture of a cute animal (not mandatory but encouraged)**

![](https://lemagdesanimaux.ouest-france.fr/images/dossiers/2021-08/manchot-empereur-102100.jpg)
